src/systemTest/java/com/flightstats/hub/system/config/GuiceModule.java

```diff
@@ -16,14 +19,21 @@
 import retrofit2.converter.scalars.ScalarsConverterFactory;
 
 import javax.inject.Singleton;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
 
 @Slf4j
 public class GuiceModule extends AbstractModule {
+    private final StringHelper stringHelper = new StringHelper();
+    private final String releaseName = "loopy-joey"; //"ddt-" + stringHelper.randomAlphaNumeric(10).toLowerCase();
     private static final String PROPERTY_FILE_NAME = "system-test-hub.properties";
 
     @Override
     protected void configure() {
+        bind(String.class).annotatedWith(Names.named("test.data")).toInstance("TEST_DATA");
```

Comment:
There is some more cleanup I can do in here, and in a few other files maybe to use the PropertyNames/PropertyLoader strategy for stuff like this